hathor/nanocontracts/context.py

```diff
@@ -108,22 +146,18 @@ def get_caller_contract_id(self) -> ContractId | None:
                 return None
             case ContractId():
                 return self.caller_id
-            case _:
+            case _:  # pragma: no cover
                 assert_never(self.caller_id)
 
-    @property
-    def timestamp(self) -> int:
-        return self.__timestamp
-
```

Comment:
We might want to keep this to ease the transition, maybe log a warning that it is deprecated.